cves/CVE-2016-4436.yml

```diff
@@ -76,12 +80,12 @@ incomplete_fix_instructions: |
   this CVE below.
 previous_fixes:
   - commit:
-    note:
+    note: The above fixes fixed the vulnerability
```

Comment:
You can keep this one note and get rid of the other comments below.

Reply:
good point thanks